llvm/include/llvm-c/DebugInfo.h

```diff
@@ -186,7 +186,30 @@ typedef unsigned LLVMMetadataKind;
 
 /**
  * An LLVM DWARF type encoding.
+ * @note Values are from DW_ATE_* constants in the DWARF specification.
```

Comment:
We don't export the Dwarf.h constants directly? Seems like that would be better (DRY priniciple).

Reply:
I don't think we don't, also this enum isn't there.

Reply:
Look at TypeKind in llvm/include/llvm/BinaryFormat/Dwarf.h. It looks like you could pull the definitions in from Dwarf.def, so we don't have to maintain them in two places.